apps/studio/components/interfaces/QueryPerformanceV2/QueryIndexes.tsx

```diff
@@ -0,0 +1,326 @@
+                      {index.schema}.{index.table}
+                    </span>
+                  </div>
+                  <span className="text-xs font-mono">{index.name}</span>
+                </div>
+              )
+            })}
+          </div>
+        )}
+      </QueryPanelSection>
+
+      <div className="border-t" />
+
+      <QueryPanelSection className="flex flex-col gap-y-6">
+        <div className="flex flex-col gap-y-2">
+          <p className="text-sm">New index recommendations</p>
+          {isLoadingChecks ? (
+            <GenericSkeletonLoader />
+          ) : !isIndexAdvisorAvailable ? (
+            <IndexAdvisorDisabledState />
+          ) : (
+            <>
+              {isLoadingIndexAdvisorResult && <GenericSkeletonLoader />}
+              {isErrorIndexAdvisorResult && (
+                <AlertError
+                  ref={project?.ref}
+                  error={indexAdvisorError}
+                  subject="Failed to retrieve result from index advisor"
+                />
+              )}
+              {isSuccessIndexAdvisorResult && (
+                <>
+                  {(index_statements ?? []).length === 0 ? (
+                    <Alert_Shadcn_ className="[&>svg]:rounded-full">
+                      <Check />
+                      <AlertTitle_Shadcn_>This query is optimized</AlertTitle_Shadcn_>
+                      <AlertDescription_Shadcn_>
+                        Recommendations for indexes will show here
+                      </AlertDescription_Shadcn_>
+                    </Alert_Shadcn_>
+                  ) : (
+                    <>
+                      {isLinterWarning ? (
+                        <Alert_Shadcn_
+                          variant="default"
+                          className="border-brand-400 bg-alternative [&>svg]:p-0.5 [&>svg]:bg-transparent [&>svg]:text-brand"
+                        >
+                          <Lightbulb />
+                          <AlertTitle_Shadcn_>
+                            We have {index_statements.length} index recommendation
+                            {index_statements.length > 1 ? 's' : ''}
+                          </AlertTitle_Shadcn_>
+                          <AlertDescription_Shadcn_>
+                            You can improve this query's performance by{' '}
+                            <span className="text-brand">{totalImprovement.toFixed(2)}%</span> by
+                            adding the following suggested{' '}
+                            {index_statements.length > 1 ? 'indexes' : 'index'}
+                          </AlertDescription_Shadcn_>
+                        </Alert_Shadcn_>
+                      ) : (
+                        <p className="text-sm text-foreground-light">
+                          Creating the following {index_statements.length > 1 ? 'indexes' : 'index'}{' '}
+                          on <code className="text-xs">public.files</code> can improve this query's
```

Comment:
Spotted this hardcoded schema name which is inaccurate. Perhaps the line should be removed?